packages/next/src/server/app-render/action-handler.ts

```diff
@@ -352,7 +354,20 @@ export async function handleAction({
           } else {
             const { parseBody } =
               require('../api-utils/node') as typeof import('../api-utils/node')
-            const actionData = (await parseBody(req, '1mb')) || ''
+
+            const nextConfig = await loadConfig(
+              PHASE_PRODUCTION_SERVER,
+              '/',
+              undefined,
+              undefined,
+              true
+            )
```

Comment:
This should be passed down in `renderOpts` as `loadConfig()` shouldn't be called for every request and only on boot-up